status-updates/2023/2023-08-17.md

```diff
@@ -0,0 +1,223 @@
+Let's name tutorials and demos Walkthroughs or Guides, collectively.
+
+TD: Your PRs on demos and tutorials are very similar, so I agree.
+
+RD: Ok, then it's settled.
+
+### Quizzes
+
+RD: They are just questions. Quizzes are collections of questions
+They can be employed by different usage components as follows:
+
+- During lectures we use slides and media for presentation and debates / discussions;
+we use questions for quizzes
+- During labs we use slides and media for presentations;
+we use guides for demos/tutorials;
+we use questions for quizzes;
+we use drills for lab work
+
+- When it comes to assignments, we use projects for homework;
+we also use drills for homework because it can be made up of multiple drills, not necessarily a project
+
+- For exams we use drills for practical exams and questions for quiz exams
+
+### Drills vs Projects
+
+RD: The only difference between them is whether or not the solution is public.
+An assignment is a collection of challenges without a public solution.
+A practice exercise does contain the solution.
+But this has to do with the implementation, not the content.
+
+RN: One thing I noted from reviewing your PRs is that everyone has developed their PRs with the OS class in mind.
+It seems that we're writing a methodology for Computer Science only.
+Please be more generic in your approaches.
+
+GM: One thing I noted while trying to port the SSS repos to OpenEdu is that we can't anticipate technical problems until we actually port some more repos.
+
+RD: That's right, but we can at least try to forsee some of them and be prepared
+
+RN: Then let's merge what we have so far and then apply further changes.
+The methodology is a volatile document, we'll keep modifying and improving it as time passes.
+
+RD: Right.
+After merging, I'll raise a new PR with an update of the methodology repo based on its own guidelines and then we'll add content to the new structure.
+
+### Difficulties when Porting SSS to OpenEdu
+
+GM: I had to break up the contents of the repository in a great number of folders.
+If someone external attempts to do the same, they will find it extremely cumbersome to create such a folder hierarchy from the GUI.
+So reorganising the content is likely going to be a difficult task.
+
+RD: This is unfortunately unavoidable.
+I'd rather offer support to any teacher attempting to port their contents to OpenEdu instead.
+It's a one-time effort, a great one, but 1 time only.
+
+GM: Furthermore, the config file for the builder is very complicated.
+I'll raise issues for the builder.
+
+RD: Git itself will be difficult to use.
+We can think of abstractions on top of it, something that simplifies the interaction via an interface.
+
+RD: We can also provide developers with an exhaustive `.yml` file from which they will have to remove what they don't want to use.
+
+SW: I'll think about a config file generator, maybe based on some regular expressions.
+
+GM + RD: This sounds like a very solid idea.
```

Comment:
To be clear, `+` means that they talk like [team rocket](https://www.youtube.com/watch?v=ZBVrPWwSlRM).